src/ontology/uberon-edit.obo

```diff
@@ -222238,6 +222238,16 @@ synonym: "fifth molar tooth" EXACT []
 is_a: UBERON:0003655 ! molar tooth
 relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
 
+[Term]
+id: UBERON:7500127
+name: otion
+def: "The most lateral anatomical point in the mastoid region." [ISBN:9780873659505]
```

Comment:
Unable to access ISBN and could not find an online reference that uses this term.
Unsure if "mastoid region" is intentionally vague or a more specific term like "mastoid process of temporal bone" should be used.